frontend/malayalam-language-explorer/style.css

```
@@ -0,0 +1,544 @@
+.malayalam-audio-btn:hover {
+  transform: translateY(-2px);
+  box-shadow: 0 10px 25px rgba(11, 107, 71, 0.6);
+  background: linear-gradient(135deg, var(--malayalam-teal), var(--malayalam-emerald));
+}
+
+.malayalam-audio-btn.playing {
+  animation: pulseEmerald 1.5s infinite;
+  background: var(--malayalam-saffron);
+}
+
+@keyframes pulseEmerald {
```

Stylelint (17.14.0)

[error] Expected keyframe name "pulseEmerald" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)